src/storage/s3.rs

```diff
@@ -16,17 +18,21 @@ mod test;
 pub(crate) use test::TestS3;
 
 pub(crate) static S3_BUCKET_NAME: &str = "rust-docs-rs";
+static S3_RUNTIME: Lazy<Mutex<Runtime>> =
+    Lazy::new(|| Mutex::new(Runtime::new().expect("Failed to create S3 runtime")));
 
 pub(crate) struct S3Backend {
     client: S3Client,
     bucket: String,
+    runtime: &'static Mutex<Runtime>,
```

Comment:
Is there a reason to make this part of `S3Backend` instead of using `S3_RUNTIME.lock().block_on()` directly?

Reply:
It was based off of [this comment](https://github.com/rust-lang/docs.rs/pull/902#issuecomment-660662482)

Reply:
Gotcha. Looking at the code I don't think it's actually necessary, we can change it to be a field as part of the switch to tokio 0.2 but for now I think it's more clear to use the `Lazy` directly.